### PR TITLE
bpo-33865: Add alias for encodings

### PR DESCRIPTION
**Pull request by tirkarthi**
Added alias for below : 

* cp720
* cp737
* cp856
* cp874
* cp875
* cp10006

Ref comment : https://bugs.python.org/msg319590
Ref for alias : https://docs.python.org/3.8/library/codecs.html#standard-encodings

Please let me know if I need to auto-generate any files or missing something.

Thanks

https://bugs.python.org/issue33865

### CODE REVIEW

**Comment by pablogsal** [comment]
Hi! Thank you for the PR. Could you add a NEWS entry and update the `standard encoding` section in `Doc/library/codecs.rst`?

**Comment by tirkarthi** [comment]
@pablogsal  I have added a NEWS entry with blurb tool and updated the docs. 

Thanks.

File: Misc/NEWS.d/next/Library/2018-06-15-12-59-55.bpo-33865.peYStB.rst
Comment (vstinner) [inline Misc/NEWS.d/next/Library/2018-06-15-12-59-55.bpo-33865.peYStB.rst]: I would say: "for code page encodings (ex: 874 for cp874)."

File: Doc/library/codecs.rst
Comment (vstinner) [inline Doc/library/codecs.rst:1027]: Can you please add ".. versionchanged:: 3.7" markup after the table to mention that you added aliases to cp720, cp737, ... ?

**Review by vstinner — CHANGES_REQUESTED** [review]

**Comment by tirkarthi** [comment]
I have made the requested changes; please review again

File: Doc/library/codecs.rst
Comment (ned-deily) [reply Doc/library/codecs.rst:1027]: @vstinner You mean "".. versionchanged:: 3.8", no?

Comment (vstinner) [reply Doc/library/codecs.rst:1027]: Oh... right :-) Sorry our 3.7 release manager!

Comment (tirkarthi) [reply Doc/library/codecs.rst:1027]: Sorry about that. Fixed with 27d9380c8738de7673de316121711665f3e8ac49 . Thanks.

**Review by vstinner — APPROVED** [review]
LGTM.

But @serhiy-storchaka proposed a different approach: https://bugs.python.org/issue33865#msg319617

@serhiy-storchaka: Are you ok with this change? I'm not 100% comfortable with your idea of modified search function. I prefer an explicit list of aliases.

**Comment by tirkarthi** [comment]
@vstinner There is a discussion about this in python-ideas and there is no clear decision on whether to assign numerical aliases or it's a problem that needs to be fixed in OP's machine. OP [confirmed](https://bugs.python.org/issue33865#msg319926) that patch related changes fixed the issue but from the python-idea's discussion I could see this to be merged only as a last resort. Maybe we can wait for some response from OP about his system configuration or a concrete decision from the discussion.

Ref : https://groups.google.com/d/msg/python-ideas/Ny1RN9wY0cI/ug6MnnEUAwAJ

**Comment by tirkarthi** [comment]
As per the discussion on the tracker this seems to be an issue with Anaconda. I am closing this PR. Thanks much @methane for the detailed analysis . Ref : https://bugs.python.org/issue33865#msg320426

**Pull request closed without merging**
